recipes/management/README.md

```diff
@@ -10,6 +10,71 @@ This is a simple, powerful way to slice our management recipes. If you are looki
+        * A captain needs to take the next village.
+* What is a PROBLEM?
+    * Three types of problems. 
+        * Inertia 
+        * ?? (things falling apart) 
+        * Power: force in a different direction than you want.
+* What are the TOOLS of winning?
+    * As an engineer:
+        * Writing code: having computers repeatedly solve the problem for you.
+        * Simplifying a design: preventing problems from happening altogether.
+        * Standardization? Making a group of problems all the same problem?
+        * Operational: doing one-off work to solve a problem (e.g. deleting files to free up disk space)
+    * As a manager:
+        * 
+    * As a manager-of-managers:
+        * 
+* For the various layers: your WINNING is their INCREASING CAPACITY?
+    * Teaching an engineer how to write better code is WINNING for an EM. But it's increasing THEIR capacity??
+    * Recursive? It does feels recursive. When you solve a problem, it removes friction/energy being used to tackle that problem, which frees up energy for other problems, meaning you've increased SOMEONE'S capacity to win...?
+    * E.g. Mashgin makes checkout easier. Frees up time for customers --> they can use that time to do more.
+    * ALL TECHNOLOGY is increasing the capacity to win for users of the technology --> is all winning translate to increasing the capacity to win for someone else?
+    * 
+* Mashgin example problems
+    * Need to integrate with P97 
+        * Increases the capacity of our sales team to win. They can close more deals!
+        * If we close one deal, we get more revenue. Increases our capacity to win since we have more $. For the customer, if they are getting value from the product, it increases their capacity to win somehow as well.
+        * Integration would be considered a win. But now assume we do NOT sell any more pilots because of it? Then we haven't actually increased the capacity of anyone, and we wasted resources doing it. So the 'win' ends up being a loss. But how could we know that ahead of time?
+        * Time value of problem solving? ROI? What's the right way to think about this? Increasing capacity is non-deterministic; just because we increased capacity doesn't mean that more wins WILL happen.
+        * There is a time component and a risk component. 
+            * E.g. a page costs me 1 hour a week to investigate & fix. If it takes me 12 hours to fix this this page, then it will pay off in 12 weeks. Fixing the page is a deterministic win: it will definitely increase my capacity to win over the next 3 months.
+            * What if the page is random? May never happen again? Or if it happens, it costs me 20 hours? Then there is a chance that in the next 3 months my 'win' would have been worthless, if it never appeared. So the 'win' might be a 'loss' (with time value included).
+        * So should we give credit to the engineering team for winning if we launch P97? Or should we wait until the pay-off happens?
```

Comment:
I'm a huge believer of incentives alignment. What will happen if you decide to delay gratification for the engineering team? In the future they'll object to any work that is uncertain... which may be what you want (you're sending a very clear message that the product story and data supporting it has to be sound and available to everyone to quesiton it) - I believe some companies work like that, FB being a big example. The risk is, if you have low confidence or you need people to maintain stuff then they may just refuse as they want to work on the big shiny projects... 
I guess it's up to you to decide how much anarchy / voting with legs you want - the more you delay / combine work with long-term winning the more you'll get :)